Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># bijoux_bd\r\n\r\n\r\naccueil.html  -barre de recherche  -lien panier -liste categories\r\npanier.html\r\n5 x categories.html  -barre de recherche -lien panier -liste produits\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	
+++ b/README.md	
@@ -1,6 +1,8 @@
 # bijoux_bd
 
 
-accueil.html  -barre de recherche  -lien panier -liste categories
-panier.html
-5 x categories.html  -barre de recherche -lien panier -liste produits
+#### accueil.html  -barre de recherche  -lien panier -liste categories
+
+#### panier.html
+
+#### 5 x categories.html  -barre de recherche -lien panier -liste produits
